k/kong/2.3.4/Dockerfile

```diff
@@ -0,0 +1,68 @@
+FROM registry.access.redhat.com/ubi7/ubi AS build
+
+WORKDIR /root
+
+ENV VERSION=2.3.4
+ENV HOME=/root
+ENV OPENSSL_DIR=/usr/local/openssl
+ENV LD_LIBRARY_PATH=/usr/local/lib
+ENV PATH=$HOME/kong/bin:/usr/local/openresty/bin:/usr/local/openresty/nginx/sbin:/usr/local/luarocks/bin:$PATH
+
+RUN yum -y install libtool zlib zlib-devel unzip zip make wget git patch make gcc-c++ ncurses-devel
+
+RUN git clone https://github.com/yaml/libyaml.git \
```

Comment:
in case we do go for ubi-8, ubi-8-codeready-builder repository has libyaml-devel already available (haven't checked on 7)